samples/Azure.AI.FormRecognizer/Generated/CodeModel.yaml

```diff
@@ -3066,6 +3066,8 @@ operationGroups:
           - !Parameter &ref_126
             schema: *ref_124
             implementation: Method
+            extensions:
```

Comment:
What motivates adding this to FormRecognizer?

Reply:
@timotheeguerin , do you know what change has added this?

Reply:
I've confirmed that this change has no effect on FormRecognizer